control-plane-operator/hostedclusterconfigoperator/controllers/resources/manifests/admissionpolicies.go

```diff
@@ -0,0 +1,43 @@
+package manifests
+
+import (
+	k8sadmissionv1beta1 "k8s.io/api/admissionregistration/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func ValidatingAdmissionPolicy(name string) *k8sadmissionv1beta1.ValidatingAdmissionPolicy {
+	return &k8sadmissionv1beta1.ValidatingAdmissionPolicy{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: k8sadmissionv1beta1.SchemeGroupVersion.String(),
+			Kind:       "ValidatingAdmissionPolicy",
+		},
+		ObjectMeta: metav1.ObjectMeta{Name: name},
+	}
+}
+
+func ValidatingAdmissionPolicyBinding(bindingName, policyName string) *k8sadmissionv1beta1.ValidatingAdmissionPolicyBinding {
+	return &k8sadmissionv1beta1.ValidatingAdmissionPolicyBinding{
+		ObjectMeta: metav1.ObjectMeta{Name: bindingName},
+		Spec: k8sadmissionv1beta1.ValidatingAdmissionPolicyBindingSpec{
+			PolicyName:        policyName,
+			ValidationActions: []k8sadmissionv1beta1.ValidationAction{k8sadmissionv1beta1.Deny},
+		},
+	}
+}
+
+func VAPolicyMatchConstraints(resources, apiVersion, apiGroup []string, operations []k8sadmissionv1beta1.OperationType) *k8sadmissionv1beta1.MatchResources {
```

Comment:
I would make this a helper function in `admissionpolicies.go` file instead, e.g. `ConstructPolicyMatchConstraints()`. Since the returned object is not a standalone resource(manifest) that can be created.